src/App.jsx

```diff
@@ -1,12 +1,6 @@
 import Homepage from './pages/Homepage';
 import './App.css';
 
-function App() {
-  return (
-    <>
-      <Homepage />
-    </>
-  );
-}
+const App = () => <Homepage />;
```

Comment:
For example: This is one of the automatically adjusted things, ESLint and Prettier will do for you. In this case you once wrote a component by creating a regular function, but all other components were written by using an arrow function.

Reply:
Oh, yes. Duly noted.